### PR TITLE
fixes #75

### LINKED ISSUE #75: Add that we are a 501c3 with our Tax id. See how Bayes impact did it (search for "2015 Bayes Impact, 501"
See http://www.bayesimpact.org

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -144,5 +144,9 @@
       %p
         = link_to 'GitHub', 'https://github.com/hackedu'
         = link_to 'Attributions', attributions_path
+    .small-12.columns.text-center
+      %p.copyright
+        © #{Date.today.year} hackEDU, a 501(c)(3) non-profit [Tax ID
```
Comment (MaxWofford) [inline app/views/pages/home.html.haml]: @zachlatta How sane is this?

**Comment by zachlatta** [comment]
Looking into this now.

**Comment by zachlatta** [comment]
:+1: in the future can you explicitly say in the title and commit title what the change does? Ex. "Add nonprofit info to homepage footer. Closes #75."

**Comment by MaxWofford** [comment]
Sure thing!

**Pull request merged**